packages/git-encrypted/src/crypto.ts

```diff
@@ -163,3 +167,35 @@ export const decryptFileContentsOnly = async ({
 
   return decryptedContents;
 };
+
+export const ensureKeysExist = async ({
+  fs,
+  gitdir,
+  encryptedDir,
+  encryptedKeysDir,
+  keyDerivationPassword,
+}: Pick<GitBaseParamsEncrypted, 'fs' | 'gitdir'> & {
```

Comment:
nit: Is `GitBAseParamsEncrypted` the most appropriate type to `Pick` from here?

I'm not sure, just jumps out at me as a question...

Reply:
Further thought, maybe the git base offline type is what we want here...

Reply:
yep I used the offline type